coderadar-graph/src/main/java/io/reflectoring/coderadar/graph/projectadministration/project/adapter/FileBaseDataMapper.java

```diff
@@ -0,0 +1,22 @@
+package io.reflectoring.coderadar.graph.projectadministration.project.adapter;
+
+import io.reflectoring.coderadar.graph.projectadministration.domain.FileEntity;
+import io.reflectoring.coderadar.projectadministration.domain.File;
+
+public class FileBaseDataMapper {
```

Comment:
Can we implement the `Mapper` interface here?

Reply:
We could. But this is only a "baseData" mapper, meaning the objects are not completely mapped. It would implement the interface but woudn't act like any of the other mappers. So I'm not sure about this. What do you think?